docs/MAVLINK_PARAMETERS.md

```diff
@@ -0,0 +1,32 @@
+# mLRS Documentation: MAVLink Parameters #
+
+([back to main page](../README.md))
+
+When enabled, the Tx module and receiver can also be configured via MAVLink parameters using a ground control station (GCS) such as MissionPlanner. This can be useful in cases in which the other methods for configuration, i.e., the CLI, Lua script, or OLED, are not (easily) accessible, such as in some stand-alone applications.
+
+Using the MAVLink parameters for configuration has its inconveniences however, due to the limitations of the MAVLink parameter system and support in GCSes.
+
+When enabled, the Tx module presents itself as MAVLink component with system id 51 (like the SiK radios) and component id 68 (= MAV_COMP_ID_TELEMETRY_RADIO). That is, it emits a [HEARTBEAT](https://mavlink.io/en/services/heartbeat.html) MAVLink message which allows a GCS to discover the component, and provides the [parameter microservice](https://mavlink.io/en/services/parameter.html). A GCS can thus connect to this component and read & write its parameters.
+
+This MAVLink component is available only under certain conditions:
+- in the Tx module the setting "Tx Mav Component" must be set to "enabled". This is usually the default setting. If not, when it must be configured using one of the other methods (CLI, Lua script, OLED). 
+- a receiver is connected for which the setting "Rx Ser Link Mode" is configured to "mavlink" or "mavlinkX".
+- if no receiver is connected then the MAVLink component is available if the last receiver which the Tx module "remembers" was configured with "Rx Ser Link Mode" = "mavlink" or "mavlinkX". This is usually the last receiver which was connected to the Tx module and for which a parameter store operation was done. 
+
+As for the other methods, if a receiver is connected, then the parameters of both the Tx module and the receiver can be configured. If no receiver is connected, then only the parameters of the Tx module can be configured. Note however that the receiver parameters are nevertheless shown.
```

Comment:
Perhaps:

"Note that receiver parameters are shown regardless of whether or not a receiver is connected."